types/global.d.ts

```diff
@@ -0,0 +1,7 @@
+// Types for compiled templates
```

Comment:
Can probably remove this?

Reply:
Yep—in fact we definitely want to! They don't get us anything useful in *most* apps/addons, but they are completely useless in this one in particular I think.